flow/envs/bottleneck_env.py

```diff
@@ -111,8 +110,8 @@ def __init__(self, env_params, sim_params, scenario, simulator='traci'):
         super().__init__(env_params, sim_params, scenario, simulator)
         env_add_params = self.env_params.additional_params
         # tells how scaled the number of lanes are
-        self.scaling = scenario.net_params.additional_params.get("scaling")
-        self.edge_dict = defaultdict(list)
+        self.scaling = scenario.net_params.additional_params.get("scaling", 1)
+        self.edge_dict = dict()
```

Comment:
is there a reason dict's are preferable here instead of defaultdict's? Just curious, I'm open to the change

Reply:
Because it wasn't using the 'default' feature of the type anywhere (plus it was the only `defaultdict` in the file so I just removed the `defaultdict` import)

Reply:
sounds good, thanks for catching that